<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>R, Reucher</author>
 <class>FileEditWidget</class>
 <widget class="QWidget" name="FileEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>188</width>
    <height>25</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="margin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="pageLineEdit">
         <layout class="QGridLayout" name="gridLayoutPageLineEdit">
          <property name="margin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLineEdit" name="lineEditFile">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageComboBox">
         <layout class="QGridLayout" name="gridLayoutPageComboBox">
          <property name="margin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QComboBox" name="comboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="toolButtonBrowse">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="icon">
         <iconset resource="qmc2.qrc">
          <normaloff>:/data/img/fileopen.png</normaloff>:/data/img/fileopen.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="toolButtonClear">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="icon">
         <iconset resource="qmc2.qrc">
          <normaloff>:/data/img/broom.png</normaloff>:/data/img/broom.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="6"/>
 <tabstops>
  <tabstop>lineEditFile</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>toolButtonBrowse</tabstop>
  <tabstop>toolButtonClear</tabstop>
 </tabstops>
 <resources>
  <include location="qmc2.qrc"/>
 </resources>
 <connections/>
</ui>
